docs/csv_data_transforms.md

```diff
@@ -1,61 +0,0 @@
-# Information for DLME CSV Data Transforms
```

Comment:
Did we delete this file for a reason?

Reply:
It was added the data providers step by step bc all of the functions apply to all methods, not just CSV

Reply:
it could be made into a generic functions page too, if that step by step is too long

Reply:
I think it's OK - I overlooked that it was included in the add a data source doc.